src/lib/components/actions.svelte

```diff
@@ -14,6 +14,7 @@
 		svg?.setAttribute('width', `${width}px`); // Workaround https://stackoverflow.com/questions/28690643/firefox-error-rendering-an-svg-image-to-html5-canvas-with-drawimage
 		if (!svg) {
 			svg = document.querySelector('#container svg');
+			svg.removeAttribute('height');
```

Comment:
I'm wondering if removing the height attribute would cause any other issues?

Reply:
@sidharthv96 

More detail I posted on issue.
When you not have `height`, browser will re-calculate `new height` from element inside `svg` element.

So, diagram will fit inside.

When edit new code, `height` will back to large number. This code block affect only for **save action**

You can test with data I put in issue

Reply:
This is what I get after downloading SVGs. Check the dimensions in bottom left.
## After fix
![image](https://user-images.githubusercontent.com/10703445/148391061-202f1338-f6e5-4da1-925d-b8d149778b7d.png)

## Before fix
![image](https://user-images.githubusercontent.com/10703445/148391112-67a70dd3-ad2a-44dd-8f5a-0d629e58ed99.png)

So this fix seems to break SVG downloads.

Also, these are 2 PNGs I downloaded (Firefox, macOS).

## After Fix
![mermaid-diagram-20220106190925](https://user-images.githubusercontent.com/10703445/148391592-0d24034e-c97f-4f09-9ae4-382ac4855387.png)

## Before Fix
![mermaid-diagram-20220106190919](https://user-images.githubusercontent.com/10703445/148391600-8d313325-c087-4e0d-a970-ae52f86f9025.png)

So is the issue OS/Browser specific?

Reply:
I'm using macOS. I'm using chrome on issue screenshot.

Now, try again with private window on Firefox 

![image](https://user-images.githubusercontent.com/535617/148402384-d6bcc02a-94d1-4098-a445-eb046bcd4487.png)

Can you see big blank space?

And here is exported png

![mermaid-diagram-20220106215724](https://user-images.githubusercontent.com/535617/148402511-94b5e53e-b9bd-4630-aeac-fb6d1fcd8488.png)

Reply:
Here is dimmension

![image](https://user-images.githubusercontent.com/535617/148402927-4d5aa033-c90a-420f-8c64-9a71cbd8cf57.png)


Maybe problem only happend on macOS?

I don't have any device with other OS here to test 😢

Reply:
Ok, so the issue is not browser/OS, but rather window size.
I was using my external monitor previously. When the window is smaller, the issue appears.

Can we brainstorm and come up with a proper solution?

![image](https://user-images.githubusercontent.com/10703445/148414548-6e4881f7-13fd-46da-8a82-fb32c563bf57.png)

Reply:
Yes, If you have any idea, we can discuss.
My solution only fix for export function. But I think problem from render function first.

Reply:
Some VSCode plugin support preview mermaid diagram have same problem, so I think it's from main mermaid.js. Not about live editor